### ISSUE TITLE
[Feature Request] Bookmark support

### ISSUE THREAD

**Issue by adworacz** [quietvoid/vspreview-rs#23]
Just like the python version has, it would be great to have bookmark support. This can be really helpful when opening the previewer days apart, saving your place (to the exact frame) that you were working on before.

This helps with everything from filter comparison to sceneing etc.

**Comment by couleurm** [comment]
could be saved as a shortcut / shell script which has arguments like --startatframe 2301

**Comment by quietvoid** [comment]
Bookmarks imply a bookmark manager or way (visually) to know what is bookmarked.
It's too complex IMO.

A CLI option for a specific frame can be done.

**Comment by adworacz** [comment]
I was thinking simply a text file with frame numbers to persist bookmarks between reloads (sounds like you already have that in part) and then a visual indicator on the progress bar to indicate where a bookmark exists, with simple arrow buttons or key combos (like page up/down) to jump between them.

Does that still sound too complex?

**Comment by quietvoid** [comment]
Yes, there's no support for visual indicators on the frame number slider.
You also still need at least keybinds for adding/removing bookmarks.
